apps/frontend/.env

```diff
@@ -11,7 +11,7 @@ NX_URL_GITHUB_VAN=https://github.com/
 NX_GA_TRACKINGID=UA-000000-01
 NX_URL_YOUTUBE_TUTPL=https://www.youtube.com/
 NX_URL_WEBSITE_KQM=https://keqingmains.com/
-NX_URL_KQM_MULTI_GUIDE=https://docs.google.com/document/d/12hrPJ3MrAF7t03gyiiVtV5uI_rUKlVlbEwl9Lf2esIQ/edit?usp=sharing
+NX_URL_KQM_MULTI_GUIDE=https://keqingmains.com/misc/multi-optimization/

```

Comment:
@frzyc we need to change the envfile repo secret. I've updated it on the drive, please update the repo value to match, thanks

also frzyc, please resolve this comment when done, then we can merge the pr

Reply:
If we leave `NX_URL_KQM_MULTI_GUIDE` unset in the repo secret it should just pull from this file